CPT331.Data.Migration/Migrations/M0002.sql

```diff
@@ -719,15 +719,15 @@ INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'C
 INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'Goldfields-Esperance District')
 INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'Great Southern District')
 INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'Kimberley District')
-INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'Metropolitan Region')
+--	INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'Metropolitan Region')	--	This looks like totals, leaving commented for the moment
 INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'Mid West-Gascoyne District')
 INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'North West Metropolitan District')
 INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'Pilbara District')
-INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'Regional Wa Region')
+--	INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'Regional Wa Region')		--	Also looks like totals
 INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'South East Metropolitan District')
 INSERT INTO [Location].LocalGovernmentArea (StateID, Name) VALUES (@@StateID, 'South Metropolitan District')
```

Comment:
I think South East equates to "East" in the image below:
![IMG Map of WA Regions](https://www.semc.wa.gov.au/PublishingImages/about-semc/the-committee/district-emergency-management-committees/District%20map%20with%20insert%20with%20borders.jpg)